source/projects/private-voice-assistant/voice_remote_classic_phone.markdown

```diff
@@ -0,0 +1,154 @@
+---
+title: "Setting up an analog phone to control Home Assistant"
+---
+
+This tutorial shows how to set up an analog phone to control Home Assistant with your voice.
+The phone is used to implement a push-to-talk mechanism. 
```

Comment:
I wouldn't quite call it a "push to talk" mechanism since you can speak more than one voice command without hanging up. Maybe something like "This allows you to have a private conversation with your smart home that only starts when you pick up the phone, and stops listening the moment you hang up."